test/helpers.ts

```diff
@@ -458,9 +460,67 @@ export const deployOldVoting = async dao => {
     return {
       schemeRegistrar,
       upgradeScheme,
-      absoluteVote
+      absoluteVote,
     };
   } catch (e) {
     console.log("deployVote failed", e);
   }
 };
+
+export const compTestHelper = async (
```

Comment:
a function with so many inputs is a sign of bad design.
first this testhelper shouldnt be here because it is only relevant for specific contract.
you need to move this back to the file where it is used.
then all the globals do not need to part of the function input. founder,staker,comp,goodfundmanager etc...
the input should be token,ctoken,tokenusdoracle,stakingamount,increasepriceofctoken maybe